packages/dcos-integration-test/extra/util/delete_ec2_volume.py

```diff
@@ -55,7 +92,7 @@ def _get_current_aws_region():
         try:
             return requests.get('http://169.254.169.254/latest/meta-data/placement/availability-zone').text.strip()[:-1]
         except requests.RequestException as ex:
-            logging.warning("Can't get AWS region from instance metadata: {}".format(ex))
+            print("Can't get AWS region from instance metadata: {}".format(ex))
```

Comment:
My personal thoughts are:
* stdout (aka print) should be reserved for functional information, like JSON output to be passed to another tool
* when dealing with context/debug information logging should be used whenever possible (i.e. in an environment where the logger will be configured)
* if the script is somewhat trivial and/or single-purposed print can be used

In this case, the script is called in isolation (either manually or by marathon) and only outputs this single message so it seems reasonable to use a print statement and avoid the somewhat verbose logging setup given that the output will just be time-stamped by marathon anyway in the test.